components/FormattedMoneyAmount.js

```diff
@@ -30,9 +32,13 @@ const FormattedMoneyAmount = ({
   currencyCodeStyles,
   isCrypto,
 }) => {
+  if (!currency) {
+    return <Span {...amountStyles}>{EMPTY_AMOUNT_PLACEHOLDER}</Span>;
+  }
```

Comment:
Decided to handle this globally since there are many places where we use the currency to display the amount (mostly from `components/expenses/ExpenseAmountBreakdown.js`), which was eventually crashing the form.

It seems reasonable to assume that an amount with currency is an invalid amount and can be treated as such.